AFQ/data.py

```diff
@@ -926,10 +930,46 @@ def s3fs_json_write(data, fname, fs=None):
         json.dump(data, ff)
 
 
-def read_mni_template(resolution=1, mask=True):
+def _apply_mask(template_img, resolution=1):
     """
+    Helper function, gets MNI brain mask and applies it to template_img.
 
-    Reads the MNI T2w template
+    Parameters
+    ----------
+    template_img : nib.Nifti1Image
+        Unmasked template
+    resolution : int, optional
+        Resolution of mask. Default: 1
+
+    Returns
+    -------
+    Masked template as nib.Nifti1Image
+    """
+    mask_img = nib.load(str(tflow.get('MNI152NLin2009cAsym',
+                                      resolution=resolution,
+                                      desc='brain',
+                                      suffix='mask')))
+
+    template_data = template_img.get_fdata()
+    mask_data = mask_img.get_fdata()
+
+    if mask_data.shape != template_data.shape:
+        mask_img = nib.Nifti1Image(
+            reg.resample(mask_data,
+                         template_data,
+                         mask_img.affine,
+                         template_img.affine),
+            template_img.affine)
+        mask_data = mask_img.get_fdata()
+
+    out_data = template_data * mask_data
+    return nib.Nifti1Image(out_data, template_img.affine)
+
+
+def read_mni_template(resolution=1, mask=True, weight=2):
```

Comment:
Let's change this to a string to match the DIPY API: `"T2w"` or `"T1w"`

```suggestion
def read_mni_template(resolution=1, mask=True, weight=2):
```